src/datascience-ui/history-react/interactivePanel.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
this seems to spuriously trigger all over in .tsx files. Should I look into it more? I just disabled in .tsx files that I used. Might be able to turn it off for all of them, but I don't know if that would hide legit errors.

Reply:
unused vars should start with _. They do in the .ts files. So I think it's legit?